cluster/config-defaults.yaml

```diff
@@ -61,6 +61,11 @@ skipper_ingress_max_replicas: "50"
 skipper_ingress_min_replicas: "3"
 skipper_ingress_cpu: "1000m"
 skipper_ingress_memory: "1Gi"
+{{if eq .Cluster.Environment "test"}}
```

Comment:
I have noticed that we mostly check `production` elsewhere but I guess there is no difference if we revert the condition to disable for `production`?..